lib/delayed_job/delayed_job_callbacks.rb

```diff
@@ -8,21 +8,20 @@ class DelayedJobCallbacks < ::Delayed::Plugin
           block.call(job)
         ensure
           duration = Time.now - start
-          Metrician.gauge("queue.process", duration) if Metrician.configuration[:queue][:process][:enabled]
-          Metrician.gauge("#{job_metric_instrumentation_name(worker)}.process", duration) if Metrician.configuration[:queue][:job_specific][:enabled]
+          Metrician.gauge("jobs.run", duration) if Metrician.configuration[:jobs][:run][:enabled]
+          Metrician.gauge("jobs.run.#{job_metric_instrumentation_name(job)}", duration) if Metrician.configuration[:jobs][:job_specific][:enabled]
```

Comment:
In person: let's fix the metric browser problem before the auto-complete problem, and update the job-specific names to match the list above.